wgpu-native/src/hub.rs

```diff
@@ -378,6 +378,7 @@ impl<B: GfxBackend> Default for Hub<B> {
 
 #[derive(Debug, Default)]
 pub struct Hubs {
+    #[cfg(any(not(any(target_os = "ios", target_os = "macos")), feature = "gfx-backend-vulkan"))]
```

Comment:
Why can't we only write `feature = "gfx-backend-vulkan"` here, like some of the other places in this commit? It seems like they should all be consistent